app/Services/Backups/InitiateBackupService.php

```diff
@@ -121,9 +136,7 @@ public function handle(Server $server, string $name = null): Backup
                 'disk' => $this->backupManager->getDefaultAdapter(),
             ], true, true);
 
-            $this->daemonBackupRepository->setServer($server)
-                ->setBackupAdapter($this->backupManager->getDefaultAdapter())
-                ->backup($backup);
+            $this->daemonBackupRepository->setServer($server)->setBackupAdapter($this->backupManager->getDefaultAdapter())->backup($backup);
```

Comment:
It would be preferable if each function call was on a newline like it was previously.